azext_iot/central/services/device.py

```diff
@@ -84,6 +85,47 @@ def list_devices(
     return devices
 
 
+def get_device_registration_summary(
```

Comment:
>get_device_registration_summary [](start = 4, length = 31)

A lot of the code in this function is copy-pasted from `list_devices`, can we functionalize the common code and reuse it? That way if we discover a bug with one function, we can fix it everywhere at the same time

Reply:
spoke offline, #wontfix